docs/querying/api.md

```diff
@@ -936,6 +936,27 @@ $ curl http://localhost:9090/api/v1/status/tsdb
 
 *New in v2.15*
 
+
+## Remote Read API
+
+This API provides data read functionality from prometheus. This interface expects [snappy](https://github.com/google/snappy) compression.
+The API definition is located [here](https://github.com/prometheus/prometheus/blob/master/prompb/remote.proto).
+
+Request are made to the following endpoint.
+```
+/api/v1/read
+```
+
+### Samples
+
+This will return a message that includes a list of raw samples.
```

Comment:
```suggestion
This returns a message that includes a list of raw samples.
```

Reply:
Does it return a message that includes a list, or does it return a list of raw samples?